src/components/CountriesList/CountriesList.tsx

```diff
@@ -22,24 +27,31 @@ export const CountriesList = () => {
 	);
 
 	if (!filteredDataByRegionandInput?.length) {
-		return <StyledParagraph>No countries found.</StyledParagraph>;
+		return (
+			<>
+				<FormPanel />
+				<StyledParagraph>No countries found.</StyledParagraph>
+			</>
+		);
 	}
 
 	return (
 		<>
+			<FormPanel />
 			{error && <p>There is an error fetching API.</p>}
 			<StyledDisplaySection>
 				{!countries && "loading"}
 				{filteredDataByRegionandInput?.map(
 					({ name, flags, population, region, capital }: CountriesProps) => (
-						<CountryCard
-							key={name.common}
-							name={name}
-							flags={flags}
-							population={population}
-							region={region}
-							capital={capital}
-						/>
+						<StyledLink to={name.common.toLocaleLowerCase()} key={name.common}>
```

Comment:
why toLocaleLowerCase instead of toLowerCase?

Reply:
fixed. some auto fill happend :)